cdk/kittyhawk/src/application.ts

```diff
@@ -115,7 +127,7 @@ export class ReactApplication extends Application {
     insertIfNotPresent(reactExtraEnv, 'PORT', props.portEnv || '80');
 
     // Configure the ingress using ingressPaths.
-    const reactIngress = [{ host: props.domain, paths: props.ingressPaths }];
+    const reactIngress = [{ host: props.domain, paths: props.ingressPaths, isSubdomain: props.isSubdomain }];
```

Comment:
The django and react configs for ingress should be the same, right? Right now they're different because of the ingressPaths thing?